src/common/App.tsx

```diff
@@ -114,22 +105,14 @@ const App: React.FC = () => {
   };
   if (dict && dict.data && dict.data.content) {
     dict.data.content = dict.data.content.replace(
-      /find\?type=(\d+)\&amp;query=(.*)/gi,
+      /find\?type=(\d+)amp;query=(.*)/gi,
```

Comment:
You need to keep \&amp; if not it won replace properly, also this logic need to move to separate layer or leave in the api layer will make more sense